src/app/calendar/calendar.hpp

```diff
@@ -10,6 +10,7 @@
 class Calendar
 {
```

Comment:
I suggest to explicitly mark private section.
It's not crucial as it is private by default, but won't do no harm and helps to read class structure faster. 
Especially, when we start the private section first, which is less common practice in my opinion.